tests/safeds_runner/server/test_websocket_mock.py

```diff
@@ -409,3 +417,51 @@ def helper_should_shut_itself_down_run_in_subprocess(sub_messages: list[str]) ->
 
 
 helper_should_shut_itself_down_run_in_subprocess.__test__ = False  # type: ignore[attr-defined]
+
+
+@pytest.mark.timeout(45)
+def test_should_accept_at_least_2_parallel_connections_in_subprocess() -> None:
+    port = 6000
+    server_output_pipes_stderr_r, server_output_pipes_stderr_w = multiprocessing.Pipe()
+    process = multiprocessing.Process(
+        target=helper_should_accept_at_least_2_parallel_connections_in_subprocess_server,
+        args=(port, server_output_pipes_stderr_w),
+    )
+    process.start()
+    while process.is_alive():
+        if not server_output_pipes_stderr_r.poll(0.1):
+            continue
+        process_line = str(server_output_pipes_stderr_r.recv()).strip()
+        # Wait for first line of log
+        if process_line.startswith("INFO:root:Starting Safe-DS Runner"):
+            break
+    connected = False
+    client1 = None
+    for _i in range(10):
+        try:
+            client1 = simple_websocket.Client.connect(f"ws://127.0.0.1:{port}/WSMain")
+            client2 = simple_websocket.Client.connect(f"ws://127.0.0.1:{port}/WSMain")
+            connected = client1.connected and client2.connected
+            break
+        except ConnectionRefusedError as e:
+            logging.warning("Connection refused: %s", e)
+            connected = False
+            time.sleep(0.5)
+    if client1 is not None and client1.connected:
+        client1.send('{"id": "", "type": "shutdown", "data": ""}')
+        process.join(5)
+    if process.is_alive():
+        process.kill()
+    assert connected
+
+
+def helper_should_accept_at_least_2_parallel_connections_in_subprocess_server(
+    port: int,
+    pipe: multiprocessing.connection.Connection,
+) -> None:
+    sys.stderr.write = lambda value: pipe.send(value)  # type: ignore[method-assign, assignment]
+    sys.stdout.write = lambda value: pipe.send(value)  # type: ignore[method-assign, assignment]
+    safeds_runner.server.main.start_server(port)
+
+
+helper_should_accept_at_least_2_parallel_connections_in_subprocess_server.__test__ = False  # type: ignore[attr-defined]
```

Comment:
Is this really needed? According to the pytest docs only functions that start with `test` are treated as tests.

Reply:
Does not seem to be needed (anymore?). I'll remove it